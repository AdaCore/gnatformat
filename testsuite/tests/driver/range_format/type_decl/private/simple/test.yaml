description: ConcreteTypeDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 102",
    "-EL 103",
    "--pipe",
  ]
