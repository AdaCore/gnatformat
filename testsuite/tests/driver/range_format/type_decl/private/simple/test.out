test.ads(<ConcreteTypeDecl ["Key"] test.ads:102:4-102:24>) - 102:1-102:24
^
   type Key is private;$

